packages/slate-tools/src/tasks/includes/messages.js

```diff
@@ -64,6 +64,16 @@ const messages = {
     return timestamp + action + amount + fileList;
   },
 
+  logDeployErrors: (cmd, files, err) => {
```

Comment:
Most of this code is the same as `logDeploys()`.  What about modifying that existing function to check for the presence of an `err`?

```js
logDeploys: (cmd, files, err) {
  const timestamp = `Deploy ${err ? 'error' : 'done'} @ ${new Date()}. `;
  const action = cmd === 'upload' ? 'added/changed ' : 'removed ';
  const amount = `${files.length} file(s): `;
  const fileList = `${files.join(', ')}.\n`;

  let logMessage = timestamp + action + amount + fileList;

  if (err) { logMessage += `${err} \n`}
  
  return logMessage;
}
```